src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/flow/FlowAnnotatorBase.java

```diff
@@ -250,7 +250,11 @@ protected void isHmerIndel(final VariantContext vc, final LocalContext localCont
             // get byte before and after
             final byte        before = getReferenceNucleotide(localContext, vc.getStart() - 1);
             final byte[]      after = getReferenceHmerPlus(localContext, vc.getEnd() + 1, MOTIF_SIZE);
-
+            if (after.length==0){
+                logger.warn("Failed to get hmer from reference, isHmerIndel and RightMotif annotations will not be calculated. " +
```

Comment:
Fixed. I actually used the logger on purpose (there is a oneshotlogger in this file already) because I thought that I would like the user to see that they are trying to call variants really close to the edge and not get this warning buried in the log. I don't expect to get millions messages like this (there are not that many edges of the contigs in a reasonable genome). 
If this is convincing, let me know and I will revert this change.